### PR TITLE
Add authorization tests for issues endpoint

### PR DESCRIPTION
**Pull request by jesusfcr**
This PR adds authorization tests for the issues endpoint.

### CODE REVIEW

File: postman/vulcan-authorization.postman_collection.json [hunk 1]
```diff
@@ -14,7 +15,6 @@
 						{
 							"listen": "test",
 							"script": {
-								"id": "3eee6672-6106-4672-b4e2-42306a86c68d",
```
Comment (jroimartin) [inline postman/vulcan-authorization.postman_collection.json:17]: Just out of curiosity. Why are all the `id` fields gone?

File: postman/vulcan-authorization.postman_collection.json [hunk 2]
```diff
@@ -1409,6 +1427,44 @@
...
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "bearer",
+							"bearer": []
+						},
```
Comment (jroimartin) [inline postman/vulcan-authorization.postman_collection.json:1452]: I'm not familiar with Postman. But, is this required? Auth seems to be set on a higher level?

```suggestion
```

**Review by jroimartin — APPROVED** [review]
LGTM. Just a couple of questions.

File: postman/vulcan-authorization.postman_collection.json (hunk 1 above)
Comment (jesusfcr) [reply postman/vulcan-authorization.postman_collection.json:17]: Probably my desktop version of postman removed the ids on export.

**Review by jroimartin — CHANGES_REQUESTED** [review]
As discussed after seeing that the tests fail because of the proposed change. It would be good to test both:

- `401` is returned when the token is missing or not valid.
- `403` is returned when the token is not authorized.

**Review by jroimartin — APPROVED** [review]
LGTM

**Review by jroimartin — APPROVED** [review]

**Pull request merged**